File: sample/kite.clj
;; Clojure Example

(def name "John")
(def age 30)

(if (>= age 18)
  (println (str "Welcome, " name))
  (println (str "Sorry, " name ". You must be 18 or older to enter.")))

(dotimes [i 5]
  (println (str "Current number: " i)))

(def fruits ["apple" "banana" "orange"])

(defn greet [name]
  (println (str "Hello, " name "!")))

(greet "Alice")

File: sample/kite.php
<?php

// Variables
$name = "John";
$age = 25;

if ($age >= 18) {
    echo "Welcome, $name!";
} else {
    echo "Sorry, $name. You must be 18 or older to enter.";
}

for ($i = 0; $i < 5; $i++) {
    echo "Current number: $i\n";
}

$fruits = array("apple", "banana", "orange");

function greet($name) {
    echo "Hello, $name!";
}

greet("Alice");

?>
